keyboards/system76/launch_1/readme.md

```diff
@@ -0,0 +1,78 @@
+# System76 Launch Configurable Keyboard (launch_1)
+
+![System76 Launch Configurable Keyboard](https://images.prismic.io/system76/b71307ac-dae6-4863-b7ca-804cd61c7ef8_launch_overhead.png?auto=compress,format&w=750)
+
+The Launch Configurable Keyboard is engineered to be comfortable, fully customizable, and make your workflow more efficient.
+
+- High-speed USB Hub
+- Works on Linux, Windows and macOS
+- 100% Open Source
+- Made in Colorado
+
+Additional Launch Keyboard resources:
+
+- Keyboard Maintainer: [System76](https://github.com/system76)
+- Hardware Supported: [System76 Launch GitHub Repository](https://github.com/system76/launch)
+- Hardware Availability: [Shop System76](https://system76.com/accessories/launch)
+
+## Building Firmware
+
+To build the firmware using `make` (after setting up the build environment), e.g.:
+
+```bash
+make -r system76/launch_1:default
+```
+
+Equivalently, using the QMK CLI:
+
+```bash
+qmk compile -kb system76/launch_1 -km default
+```
+
+## Flashing Firmware (DFU)
+
+To build and flash the firmware on the keyboard, e.g.:
+
+```bash
+make -r system76/launch_1:default:flash
+```
+
+Equivalently, using the QMK CLI:
+
+```bash
+qmk flash -kb system76/launch_1 -km default
+```
+
+## Flashing Firmware (ISP)
+
+To flash the firmware using ISP refer to the [_ISP Flashing Guide_](https://docs.qmk.fm/#/isp_flashing_guide).
+
+Build the firmware and bootloader with:
+
+```bash
+make -r system76/launch_1:default:production
+```
+
+Run `avrdude` to flash the fuses:
+
+```bash
+avrdude -c <programmer> -p atmega32u4 -U lfuse:w:0x5E:m -U hfuse:w:0xD9:m -U efuse:w:0xCB:m
+```
+
+Run `avrdude` to flash the QMK firmware and Atmel/Microchip bootloader into the ROM:
+
+```bash
+avrdude -c <programmer> -p atmega32u4 -U flash:w:.build/system76_launch_1_default_production.hex
+```
+
+## Environment Setup
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. If new to QMK, start with the [_Complete Newbs Guide_](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+- **Bootmagic reset**: Hold down the key at (0,0) in the matrix (Escape) and plug in the keyboard.
+- **Keycode in layout**: Press the key mapped to `RESET` in the second layer (Escape).
```

Comment:
This does not seem to describe the actual `RESET` keycode handling for this keyboard; see the comment for the corresponding code in `process_record_kb()`.

Reply:
That is correct. Using the `RESET` keycode on the actual hardware puts the keyboard into a special "unlocked" RGB mode, but I do not think it jumped into the bootloader. Neither QMK CLI nor QMK toolbox, nor Linux or Windows seem to detect it as such. Only Bootmagic and shorting the pads work to get it into that mode.

Reply:
That sounds like it would subvert users' expectations for how the `RESET` keycode works in pretty much every other QMK keyboard. The default implementation for this board should be as simple as possible and follow the general QMK conventions, with perhaps a vendor keymap that contains all of the System76-specific functionality.

Reply:
I agree, I tried to use it and didn't do what I expected. However, I honestly would say this is a bug or something missed in verification rather than a deliberate attempt to circumvent basic QMK functionality.

Reply:
Mentioning System76 folks in case they care to comment on this or any other of the updates: @jackpot51, @leviport, @ids1024

Reply:
Fixed this by guarding the RESET HID command behind a preprocessor define. This appears to be the least impact change to the existing code as the Keyboard Configurator does not appear to use it.